app/models/order_detail.rb

```diff
@@ -0,0 +1,5 @@
+class OrderDetail < ApplicationRecord
+  belongs_to :order
+  belongs_to :item, foreign_key: :item_id,
```

Comment:
sao k để là product luôn mà phải để tên là item thế em

Reply:
Tại e để cái foreign_key trong OrderDetail là item_id r :3 nên giờ để cái tên association là item cho nó đồng nhất :v

Reply:
sửa lại là từ migration rồi rollback xong migrate lại đi thanh niên, chứ để như nà tù lắm, sau gọi product toàn phải gọi bằng key item ấy =))